### PR TITLE
Publish to GitHub pages

### PR DESCRIPTION
**Pull request by frankieroberto**

### CODE REVIEW

**Comment by frankieroberto** [comment]
Minimal publishing of the README as an HTML page using the eleventy plugin with GitHub Pages.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "build": "eleventy --quiet",
+    "start": "eleventy --serve --quiet"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@11ty/eleventy": "^2.0.1",
+    "@x-govuk/govuk-eleventy-plugin": "^5.0.3",
+    "eleventy": "^1.0.6"
```
Comment (paulrobertlloyd) [inline package.json]: Not sure dependency is needed (or even correct)

```suggestion
    "eleventy": "^1.0.6"
```

File: .eleventy.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    footer: {
+      contentLicence: {
+        html: 'Licensed under the <a class="govuk-footer__link" href="https://github.com/x-govuk/govuk-prototype-components/blob/main/LICENSE.txt">MIT Licence</a>, except where otherwise stated'
+      },
+      copyright: {
+        text: '© X-GOVUK'
+      },
+      meta: {
+        items: []
+      }
```
Comment (paulrobertlloyd) [inline .eleventy.js]: Don’t need this property if empty.

```suggestion
      meta: {
        items: []
      }
```

**Review by paulrobertlloyd — APPROVED** [review]
A few tweaks, but otherwise looks great. We could add the common SCSS settings file, but not sure that’s needed given the small scale of this page.

File: package.json (hunk 1 above)
Comment (frankieroberto) [reply package.json]: Opps, not sure how that snuck in.

**Pull request merged**